CalibFormats/SiStripObjects/interface/SiStripDetInfo.h

```diff
@@ -23,7 +23,7 @@
 // user include files
 #include <map>
 #include <vector>
-
+#include <stdint.h>
```

Comment:
Shouldn't this be
```suggestion
#include <cstdint>
```
(as in #34077)?